src/containers/DidDocumentView/DidDocumentView.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { Contact } from '../../types/Contact'
+import './DidDocumentView.scss'
+import Code from '../../components/Code/Code'
+
+type Props = {
+  did: Contact['did']
+  children: any
+}
+
+const DidDocumentView = (props: Props) => {
+  const { did } = props
+  return (
+    <>
+      <div className="attributes">
```

Comment:
Question:
Did you see my comment on the previous review? Looks like you resolved it, but it's unchanged.

Reply:
Ok, Iterated again.